### DISCUSSION TITLE
Separate package for renovate-config-validator

### DISCUSSION THREAD

**Discussion by ypresto** [renovatebot/renovate#10394] [Suggest an Idea]
I just want to validate config file, but `npx --package renovate -c 'renovate-config-validator'` installs all dependencies necessary to run bot.

**Comment by rarkins** [comment]
Even if we were to publish the validator as a separate package, the most obvious way to do so would be to use `renovate` as a dependency, which would still result in `npx` installing all dependencies. Hence to achieve what you *really* seem to want, which is quicker validator install, requires more than just a separate package.
